README.md

```diff
@@ -1,15 +1,15 @@
 [![Build Status](https://travis-ci.org/gnosis/dex-contracts.svg?branch=master)](https://travis-ci.org/gnosis/dex-contracts?branch=master)
 [![Coverage Status](https://coveralls.io/repos/github/gnosis/dex-contracts/badge.svg?branch=master)](https://coveralls.io/github/gnosis/dex-contracts?branch=master)
 
-# dFusion - Smart Contracts
+# Gnosis Protocol - Smart Contracts
 
-The **dFusion Exchange** is a fully decentralized trading protocol which facilitates ring trades via discrete auction between several [ERC20](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md) token pairs.
+The **Gnosis Protocol Exchange** is a fully decentralized trading protocol which facilitates ring trades via discrete auction between several [ERC20](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md) token pairs.
```

Comment:
Should this be `**Batch Exchange**` as that is the name of the smart contract?

Reply:
I feel like here it's referring more to the name of the overall project, thus I'd prefer Gnosis Protocol but no strong feelings.